qiskit/__init__.py

```diff
@@ -62,8 +62,7 @@
 if sys.version_info[0] == 3 and sys.version_info[1] == 6:
     warnings.warn(
         "Using Qiskit with Python 3.6 is deprecated as of the 0.17.0 release. "
-        "Support for running Qiskit with Python 3.6 will be removed in a "
-        "future release.",
+        "Support for running Qiskit with Python 3.6 will be removed in Terra 0.20.0.",
```

Comment:
Sigh didn't his the save button before the review button and my comment was lost.

Maybe we should reword the message to be something like "..deprecated as of the qiskit-terra 0.17.0 release...removed in the qiskit-terra 0.20.0 release." just to make it clear we're talking about terra package versions and not the metapackage version.

Reply:
Yep, certainly doesn't hurt.